addons/cargo/stringtable.xml

```diff
@@ -498,6 +498,12 @@
             <Czech>Upravuje jak dlouho výsadek trvá pro náklad.</Czech>
             <Spanish>Modificador de tiempo de lanzamiento en paracaídas de un objeto cargado</Spanish>
         </Key>
+        <Key ID="STR_ACE_Cargo_unloadOnKilled">
+            <English>Unload Cargo On Killed</English>
+        </Key>
+        <Key ID="STR_ACE_Cargo_unloadOnKilled_description">
+            <English>The chance for each cargo loaded into a vehicle to be unloaded if the vehicle is killed.</English>
```

Comment:
if it's only going to run on objects and not classnames//strings, then we should add some kind of note

```suggestion
            <English>The chance for each cargo loaded into a vehicle to be unloaded if the vehicle is killed.</English>
```
```suggestion
            <English>The chance for each cargo loaded into a vehicle to be unloaded if the vehicle is killed. [Only Objects]</English>
```